### ISSUE TITLE
ignore the namespace_id_not_found in the alba disk-safety test.

### ISSUE THREAD

**Issue by jeroenmaelbrancke** [openvstorage/openvstorage-health-check#314]
## Problem description
Ignore this error.
```
CRIT - EXCEPTION HC000 - Could not fetch alba information for backend nvmebackend Message: Command 'get-disk-safety' failed with 'Namespace manager exception: Nsm_model.Err.Namespace_id_not_found'.
```

**Comment by JeffreyDevloo** [comment]
This is actually quite hard to catch as the get-disk-safety returns a list of namespaces. It's not that we fetch each namespace one by one.

**Comment by jeroenmaelbrancke** [comment]
On OVH we forgot to restart the proxies after updating to 1.3.7.
Closing the tickets and if i still see this issue i will open the tickets again.

**Comment by jeroenmaelbrancke** [comment]
Problem is not solved. This night we received an urgent with the same error.
Alba version 1.3.8

**Comment by wimpers** [comment]
@domsj  is this error still valid and what can be causing this?

**Comment by domsj** [comment]
fixed (second attempt) as of alba 1.3.9, so I think this ticket can be closed
(first attempt was in alba 1.3.7)
